src/mrack/providers/provider.py

```diff
@@ -181,7 +181,7 @@ def parse_error_hosts(self, hosts):
         logger.debug(f"{self.dsp_name}: Checking provisioned hosts for errors")
         for host in hosts:
             logger.debug(f"{self.dsp_name}: Host - {host.id}\tStatus - {host.status}")
-            if host.status == STATUS_ERROR:
+            if host.status in (STATUS_ERROR, STATUS_TIMED_OUT):
```

Comment:
I wonder if we should invert the logic here and check for statuses that we **know** are not considered errors. Just to be "future proof". Does it make sense?

Reply:
I jump on this right now